### DISCUSSION TITLE
PLACING TILES

### DISCUSSION THREAD

**Discussion by Lunaire333** [AmyE123/TactiKit#5] [Map Editor Feedback]
I love that you must use the left mouse button to create a tile on the plain canvas. 

However, what would be better is to be able to create tiles by also dragging the left mouse instead of constantly clicking on the chosen square.
This will not only avoid tiredness from the finger constantly clicking, but it will also not make a constant noise coming from the mouse, which might bother the user.

![PLACING TILES](https://github.com/AmyE123/TactiKit/assets/158181584/bf269683-c35b-4ac3-a3da-9a4b59203665)

**Comment by Jiunnyih-Tan** [comment]
Also, the ability to highlight and edit multiple tiles.

**Comment by UP778454** [comment]
I think the use of Shift and then click dragging might better serve user experience by keeping it simpler and more separated
